server/src/main/java/io/crate/execution/ddl/tables/AddColumnTask.java

```diff
@@ -225,4 +238,11 @@ private static void mergeDeltaIntoExistingMapping(Map<String, Object> existingMa
             }
         );
     }
+
+    /**
+     * @return **only** new columns (with assigned OID-s).
+     */
```

Comment:
```suggestion
    /**
     * @return columns added by the last {@link #execute} call. Columns part of the {@link AddColumnRequest} which already existed are not included.
     */
```

And add a `addedColumns.clear()` in the start of `.execute` to make sure it could be called multiple times.
Currently not necessary given how it's used, but to match the docs. It's easier to explain this way than to say that it can only be used once.

Reply:
Added `clear()` in https://github.com/crate/crate/pull/14617/commits/022a78d11d817b956b165d5865e27801a763c0c2